Makefile.dapper

```diff
@@ -28,6 +28,10 @@ shell: .dapper
 # This only lists targets accessible inside dapper (which are 99% of targets we use)
 targets:
 	@$(RUN_IN_DAPPER) make -pRrq : 2>/dev/null |\
-		grep -oP '^(?!Makefile.*)[-[:alnum:]]*(?=:)' | sort -u
+		grep -oP '^(?!Makefile.*)[-[:alnum:]]*(?=:)' | sort -u > /tmp/targets.txt;\
+		for target in $$(cat /tmp/targets.txt); do\
+			printf "$${target}\t\t";\
```

Comment:
To align the descriptions consistently, this works better:

```suggestion
			printf "%-24s" "$${target}";\
```

Reply:
I knew there must be a better way but I'm not that familiar with all the Linux command :)

Reply:
This is C really :stuck_out_tongue: https://pubs.opengroup.org/onlinepubs/9699919799/functions/snprintf.html